client/my-sites/plans/jetpack-plans/product-store/featured-item-card/index.tsx

```diff
@@ -15,13 +16,16 @@ export const FeaturedItemCard: React.FC< FeaturedItemCardProps > = ( {
 	isCtaDisabled,
 	isCtaExternal,
 	isProductInCart,
+	isVertical,
 	onClickCta,
 	price,
 	title,
 	variant,
 } ) => {
 	return (
-		<div className="featured-item-card">
+		<div
+			className={ clsx( 'featured-item-card', { [ 'featured-item-card--vertical' ]: isVertical } ) }
```

Comment:
I'm keeping both variants in case the designs for the mid-range breakpoints use it. I will adjust after the final design review.

Reply:
I decided to keep the vertical variant separation from the original implementation using the additional class.